training/tf/parse.py

```diff
@@ -35,17 +35,14 @@
 import time
 import threading
 
-# 16 planes, 1 side to move, 1 x 362 probs, 1 winner = 19 lines
-DATA_ITEM_LINES = 16 + 1 + 1 + 1
-
 # Sane values are from 4096 to 64 or so. The maximum depends on the amount
 # of RAM in your GPU and the network size. You need to adjust the learning rate
 # if you change this.
 BATCH_SIZE = 512
 
-# Use a random sample of 1/16th of the input data read. This helps
-# improve the spread of games in the shuffle buffer.
-DOWN_SAMPLE = 16
+# Use a random sample input data read. This helps improve the spread of
+# games in the shuffle buffer.
+DOWN_SAMPLE = 2
```

Comment:
Are good values of this format dependent?

Reply:
No, they're dependent on the amount of shuffling being done. The data shuffle in v2_write_training combined with the data shuffle in parse.py is enough that down sampling isn't required to get good move diversity in mini-batches.